packages/admin-rte/src/core/Controls/BlockTypesControls.tsx

```diff
@@ -39,7 +34,39 @@ export default function BlockTypesControls({
                         {c.label}
                     </MenuItem>
                 ))}
-            </sc.Select>
+            </Select>
         </FormControl>
     );
 }
+
+export type CometAdminRteBlockTypeControlsClassKeys = "root" | "select";
+
+const styles = (theme: Theme) =>
+    createStyles<CometAdminRteBlockTypeControlsClassKeys, any>({
+        root: {},
+        select: {
+            color: theme.rte.colors.buttonIcon,
+            minWidth: 180,
+            lineHeight: "24px",
+            fontSize: 14,
+            padding: 0,
+            "& [class*='MuiSvgIcon-root']": {
+                color: "inherit",
+            },
+        },
+    });
+
+const StyledBlockTypesControls = withStyles(styles, { name: "CometAdminRteBlockTypeControls" })(BlockTypesControls);
+
+// If there are no dropdown-features, this must return null not just an empty component, to prevent an empty item from being rendered in Toolbar
+export default (p: IControlProps) => {
+    const { editorState, setEditorState, editorRef, options } = p;
+    const { supports: supportedThings, blocktypeMap } = options;
+    const blockTypes = useBlockTypes({ editorState, setEditorState, supportedThings, blocktypeMap, editorRef });
+
+    if (!blockTypes.dropdownFeatures.length) {
```

Comment:
This is fine. But in reality I think this can never happen, there is always at least one option (unstyled-block), or am I wrong?

Reply:
or should it be `if (!blockTypes.dropdownFeatures.length > 1) {` to show the dropdown only when there is something to choose from?

Reply:
`dropdownFeatures` does not include `unstyled`, only additional options.  
When there are no additional options (only `unstyled`) this will return `null` and therefore hide the select.